notebooks/image_models/solutions/create_tfrecords_at_scale.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>* Too many "In this lab": "dataset. In this lab you" --&gt; "dataset, and you"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done.</p>